go/go.wheezy.bash

```diff
@@ -15,11 +15,16 @@
 # limitations under the License.
 . /builder/prepare_workspace.inc
 prepare_workspace || exit
-cat <<EOF
-**DEPRECATION WARNING**
-The 'gcr.io/cloud-builders/go:wheezy' image is deprecated. Please switch go 'gcr.io/cloud-builders/go:debian'.
-Starting no earlier than 2017-12-12 the 'gcr.io/cloud-builders/go:wheezy' image will be broken and no longer work.
+
+
+cat << EOF
+*******************************************************************************
+*** NOTICE: The 'go:wheezy' builder is deprecated. Use 'go:debian' instead. ***
+*** Starting no earlier than December 12, 2017, this builder will be broken ***
```

Comment:
Isn't this a very short notice?

Reply:
The deprecation warning has been there in a different form since #132 which was merged September 12